src/main/resources/db/changelog/db.changelog-master.xml

```diff
@@ -6,9 +6,9 @@
         xsi:schemaLocation="http://www.liquibase.org/xml/ns/dbchangelog
```

Comment:
master changelog should just import other changelogs, it shouldn't contain changesets